### PR TITLE
adding jquery-validation

### PR DESCRIPTION
**Pull request by dshovchko**

### CODE REVIEW

File: src/scripts/form/form.js [hunk 1]
```diff
@@ -85,8 +93,14 @@ var FormView = Marionette.LayoutView.extend({
     },
     
     onSubmit: function () {
+        console.log("check", this.$("#movieForm").valid());
```
Comment (baksofill) [inline src/scripts/form/form.js]: do we need it?

File: src/scripts/form/form.js [hunk 2]
```diff
@@ -85,8 +93,14 @@ var FormView = Marionette.LayoutView.extend({
...
+        if (!this.$("#movieForm").valid()) {
+            console.log("not valid");
```
Comment (baksofill) [inline src/scripts/form/form.js]: do we need it?

File: src/scripts/form/form.js [hunk 3]
```diff
@@ -85,8 +93,14 @@ var FormView = Marionette.LayoutView.extend({
...
+            return;
+        }
         var data = {};
         this.els.forEach(function(el) {
+
```
Comment (baksofill) [inline src/scripts/form/form.js]: please remove empty line

File: src/scripts/handsOnTable/editors/input.js [hunk 4]
```diff
@@ -2,23 +2,31 @@ var Handsontable = require("handsontable");
...
 var InputEditor = Handsontable.editors.BaseEditor.prototype.extend();
 InputEditor.prototype.init = function () {
     this.div = document.createElement("DIV");
     this.div.style.display = "none";
-
-    this.iv = new InputView({ options: { key: null, value: null } });
-
-    // Attach node to DOM, by appending it to the container holding the table
-    this.div.appendChild(this.iv.el);
+    this.form = document.createElement("FORM");
```
Comment (baksofill) [inline src/scripts/handsOnTable/editors/input.js]: is it ok to use letters in uppercase?

File: src/scripts/handsOnTable/editors/input.js [hunk 5]
```diff
@@ -37,16 +45,27 @@ InputEditor.prototype.open = function () {
     this.div.style.left = tdOffset.left - rootOffset.left + "px";
     this.div.style.zIndex = 9999;
     this.div.style.height = height + "px";
-    this.div.style.minWidth = width + "px";
+    //this.div.style.minWidth = width + "px";
```
Comment (baksofill) [inline src/scripts/handsOnTable/editors/input.js]: do we need it?

File: src/scripts/handsOnTable/editors/obj.js [hunk 6]
```diff
@@ -1,14 +1,18 @@
...
+ * @editor ObjEditor
+ * @class ObjEditor
+ */
 var ObjEditor = Handsontable.editors.BaseEditor.prototype.extend();
 ObjEditor.prototype.init = function () {
     this.div = document.createElement("DIV");
     this.div.style.display = "none";
-
-    this.schemaProps = schema.getPropertiesAsArray();
+    this.form = document.createElement("FORM");
```
Comment (baksofill) [inline src/scripts/handsOnTable/editors/obj.js]: is it ok to use letters in uppercase?

**Comment by dshovchko** [comment]
I have made changes in form.js and handsontable editors

**Pull request merged**